frontend/package.json

```diff
@@ -11,10 +11,11 @@
   "dependencies": {
     "@mdi/font": "^7.4.47",
     "@mdi/js": "^7.4.47",
-    "axios": "^1.6.7",
+    "axios": "^1.6.8",
     "chart.js": "^4.4.2",
     "swiper": "^11.1.1",
     "vue": "^3.4.15",
+    "vue-jwt-decode": "^0.1.0",
```

Comment:
This seems to be a 6 year old library for vue 2, use a different library that works with vue 3

Reply:
Fixed